packages/quick_actions/lib/quick_actions.dart

```diff
@@ -54,9 +66,14 @@ class QuickActions {
       assert(call.method == 'launch');
       handler(call.arguments);
     });
+    // ignore: deprecated_member_use_from_same_package
     runLaunchAction(handler);
   }
 
+  /// This is only exposed for the unit tests. Do not rely on it, it may break
+  /// without warning in the future.
+  @visibleForTesting
+  @deprecated
```

Comment:
@collinjackson @diesersamat as far as I can tell this method isn't intended to be part of the public API, but I'm not sure if I'm misunderstanding the original intent here. From what I could tell reading through the codebase this looks like it's purely a helper method for `intialize`, and users of the plugin are never expected to call this on its own. I think it was originally added to help with testing. If my reading is wrong, what's the intended use case of this and calling `getLaunchAction` outside of `initialize`? It was added in flutter/plugins#1800. if that helps.